h/celery.py

```diff
@@ -34,7 +34,7 @@
     task_acks_late=True,
     worker_disable_rate_limits=True,
     task_ignore_result=True,
-    imports=("h.tasks.admin", "h.tasks.cleanup", "h.tasks.indexer", "h.tasks.mailer"),
+    imports=("h.tasks.cleanup", "h.tasks.indexer", "h.tasks.mailer"),
```

Comment:
I think this is a definite sign we are going in the right direction. This was absolutely an indexing task, not an "admin" task, and this is one less spaghetti arm of the indexing behavior cut off.